lib/HTTP/Tiny.pm

```diff
@@ -115,7 +117,7 @@ sub new {
         max_redirect => 5,
         timeout      => defined $args{timeout} ? $args{timeout} : 60,
         keep_alive   => 1,
-        verify_SSL   => $args{verify_SSL} || $args{verify_ssl} || 0, # no verification by default
+        verify_SSL   => $args{verify_SSL} // $args{verify_ssl} // _verify_SSL_default(),
```

Comment:
That's an interesting change in behavior. Consider:
```perl
my $http = HTTP::Tiny->new(
    verify_SSL => 0,
    verify_ssl => 1,
);
```
With the old code, this turns host verification on. With this change, this turns host verification **off**.